doc/tech/README.md

```diff
@@ -144,7 +144,8 @@ There will be dedicated OVP hacking tracks to facilitate VNF vendor onboarding a
 
 The CNTT will collaborate with the respective API workgroups of SDOs (ETSI, MEF, TM Forum) as much as possible.  However, to collate on the relevant APIs from these SDOs in some cases requires special permission since information might not be available to the public.  For example. MEF LSO APIs & TM Forum OpenAPIs are accessible by members only.
 
->_More diagrams needed to illustrate relationships_
+<p align="center"><img src="./figures/tech_relation.png" alt="scope" title="Document Types" width="100%"/></p>
```

Comment:
- I think we should not show here technology choices, like Airship
- Please provide the source of the figure too :)

Reply:
@CsatariGergely I think we are mentioning opensource technologies and tools in various places.

Reply:
@CsatariGergely fixed in https://github.com/cntt-n/CNTT/pull/1433/commits/252b7178e8c9e3e41513a70896714dec77fdf563